redundant-bmc/src/active_role_handler.cpp

```diff
@@ -165,4 +165,55 @@ auto ActiveRoleHandler::getFailoverBlockedReason(
     co_return fo_blocked::Reason::bmcNotPassive;
 }
 
+// NOLINTNEXTLINE
+sdbusplus::async::task<> ActiveRoleHandler::failoverStartActiveTarget()
+{
+    try
+    {
+        co_await providers.getServices().startUnit(bmcActiveTarget);
+    }
+    catch (const sdbusplus::exception_t& e)
+    {
+        // TODO: error log
+        lg2::error(
+            "Failed while starting BMC active target during failover: {ERROR}",
+            "ERROR", e);
+    }
+}
+
+// NOLINTNEXTLINE
+sdbusplus::async::task<> ActiveRoleHandler::failoverWaitForSibling()
+{
+    // Wait a bit to ensure the sibling reset is detected before we
+    // start waiting for it to come back. After the active target
+    // has real code in it that runs longer than a few seconds,
+    // this could be removed.
+    co_await providers.getSibling().pauseForHeartbeatChange();
+
+    // Wait for the sibling heartbeat to come back
+    co_await providers.getSibling().waitForSiblingUp();
+
+    // Wait for the sibling to get to Ready state so redundancy can
```

Comment:
Just a suggestion: 

shall we mention like wait for sibling steady state to determine redundancy again?

Reply:
I’m a bit confused about the difference between Ready and Steady states. I initially thought redundancy determination occurs once the system reaches the Ready state, but I believe it actually happens only when it reaches the Steady state.

Reply:
OK I see what you mean now.  I will update the comment to always say steady state, since if it hits Quiesced we will stop waiting.

Reply:
Updated.